constants.production.yaml

```diff
@@ -0,0 +1,44 @@
+genesisN: 1000
+maxLocalTxs: 10000
+defaultPeers: []
+mdNoBlocksSlotThreshold: 1
+mdNoCommitmentsEpochThreshold: 3
+protocolMagic: 0
+enhancedMessageBroadcast: 2
+blockRetrievalQueueSize: 100
+recoveryHeadersMessage: 100 # should be more than k, less than blockRetrievalQueueSize
+kademliaDumpInterval: 4
+
+# Fundamental constants for core protocol
+k: 6
+slotDurationSec: 20
+
+# Time slave/lord
+sysTimeBroadcastSlots: 6
+
+# Networking
+neighboursSendThreshold: 4
+networkDiameter: 15
+networkConnectionTimeout: 2000
+networkReceiveTimeout: 60000
+
+# GodTossing
+vssMaxTTL: 10 # epochs
+vssMinTTL : 2 # epochs
+mpcSendInterval: 12 # must be less than (k * slotDuration - networkDiameter)
+mpcThreshold: 0.01 # 1% of stake
+
+# Delegation
+delegationThreshold: 0.005 # 0.5% of stake
+maxBlockProxySKs: 10000
+ntpResponseTimeout: 1000000 # 1 sec
+ntpPollDelay: 300000000 # 300 sec
+
+# Update mechanism
+updateServers: []
+updateProposalThreshold: 0.1 # 10% of total stake
+updateVoteThreshold: 0.001 # 0.1% of total stake
+updateImplicitApproval: 40000 # slots
+
+# Production start time
+productionNetworkStartTime: 0
```

Comment:
This file was removed from master. Probably merge conflict resolved inappropriately. Need to remove it here as well